### ISSUE TITLE
Zindex are not reflected in collaboration

### ISSUE THREAD

**Issue by christophepas** [excalidraw/excalidraw#2613]
Here is a test ran with 2 clients in parallel, sending an element all the way  back in client 1 doesn't reflect in client 2. 

![CleanShot 2020-12-19 at 11 04 25](https://user-images.githubusercontent.com/6739099/102686869-e8decb80-41ea-11eb-8829-2bd054ee2e88.gif)

**Comment by dwelle** [comment]
Duplicate of https://github.com/excalidraw/excalidraw/issues/1376 — as a matter of fact I discussed this just yesterday, and it definitely needs to be resolved soon. It's not exactly trivial to do this with our new reconciliation algo while not compromising on speed, but truth be told I haven't put much though into this yet. The linked issue contains requirements we need to meet.

**Comment by christophepas** [comment]
Thanks for the swift reply, indeed a tricky one.
